lib/collections.js

```diff
@@ -15,9 +15,9 @@ var paragraph =
 // The word list normalized (all lower case with punctuation removed)
 var normalizedWords = [];
 
-/* your code starts here */
+normalizedWords = paragraph.toLowerCase().replace(/[^\w']+/g, " ").split(/\s+/);
```

Comment:
This is mostly fine, BUT there's an issue that this can cause if we end in a special character - in that case, `split` will give us an extra empty string after place where the special character used to be. And in fact, that's what will happen with `paragraph` here, since it ends in a period.